lmdeploy/pytorch/engine/engine_instance.py

```diff
@@ -42,10 +42,8 @@ async def async_try_add_session(req_sender: RequestSender, session_id: int):
 
 async def async_end(req_sender: RequestSender, session_id: int):
     """End the given session."""
-    resp = await req_sender.async_send(RequestType.END_SESSION,
-                                       dict(session_id=session_id))
-    _check_resp_success(resp, (f'Failed to end session: {session_id}. '
-                               f'Error: {resp.type}.'))
+    await req_sender.async_send_async(RequestType.END_SESSION,
+                                      dict(session_id=session_id, resp=False))
```

Comment:
should it be `response=False` as used in https://github.com/InternLM/lmdeploy/blob/5d6c4ab39017d3e1e3e5f2c9edb2f34859e36ee5/lmdeploy/pytorch/engine/engine.py#L248